src/TipBox.vue

```diff
@@ -209,4 +212,16 @@
     .font-black {
         color: #24292e;
     }
+
+    .icon-extra-large {
```

Comment:
Specify `.icon-extra-large i` to not affect other elements.

Though as I mentioned in the comments that I linked earlier, such arbitrarily styling should really be in the site's own CSS file, rather than included in MarkBind/vue-strap.

Reply:
> Though as I mentioned in the comments that I linked earlier, such arbitrarily styling should really be in the site's own CSS file, rather than included in MarkBind/vue-strap.

Oh, I see. I misunderstood thinking it was only about the usage of "icon-large" instead. Will make the change.